buildkite/pipelines.go

```diff
@@ -20,6 +20,8 @@ type CreatePipeline struct {
 	Steps      []Step `json:"steps" yaml:"steps"`
 
 	// Optional fields
+	Configuration                   string            `json:"configuration,omitempty" yaml:"configuration,omitempty"`
```

Comment:
With YAML steps enabled, you supply either Steps or Configuration, so I believe these both need to be optional now:

```suggestion

	// Optional fields
	Steps                           []Step            `json:"steps,omitempty" yaml:"steps,omitempty"`
	Configuration                   string            `json:"configuration,omitempty" yaml:"configuration,omitempty"`
```